mteb/evaluation/MTEB.py

```diff
@@ -513,7 +524,7 @@ def run(
 
             try:
                 task.check_if_dataset_is_superseded()
-                task.load_data(eval_splits=task_eval_splits, **kwargs)
+                task.load_data(**kwargs)
```

Comment:
Could you please help me understand why this is not needed anymore?

Reply:
It is not used in dataset loading